src/test/java/uk/gov/companieshouse/filinghistory/api/service/ResolutionPutRequestValidatorTest.java

```diff
@@ -0,0 +1,268 @@
+                                RequestBodyTestArgument.builder()
+                                        .modifyEntityId(null)
+                                        .build())),
+                Arguments.of(
+                        Named.of("Empty entity ID",
+                                RequestBodyTestArgument.builder()
+                                        .modifyEntityId("")
+                                        .build())),
+                Arguments.of(
+                        Named.of("Null delta at",
+                                RequestBodyTestArgument.builder()
+                                        .modifyDeltaAt(null)
+                                        .build())),
+                Arguments.of(
+                        Named.of("Empty delta at",
+                                RequestBodyTestArgument.builder()
+                                        .modifyDeltaAt("")
+                                        .build())),
+                Arguments.of(
+                        Named.of("Null company number",
+                                RequestBodyTestArgument.builder()
+                                        .modifyCompanyNumber(null)
+                                        .build())),
+                Arguments.of(
+                        Named.of("Empty company number",
+                                RequestBodyTestArgument.builder()
+                                        .modifyCompanyNumber("")
+                                        .build()))
+        );
+    }
+
+    private record RequestBodyTestArgument(InternalFilingHistoryApi requestBody) {
+
+        private static RequestBodyTestArgumentBuilder builder() {
+            return new RequestBodyTestArgumentBuilder();
+        }
+
+        private static class RequestBodyTestArgumentBuilder {
+
+            private final InternalFilingHistoryApi requestBody;
+
+            public RequestBodyTestArgumentBuilder() {
+                requestBody = new InternalFilingHistoryApi()
+                        .externalData(new ExternalData()
+                                .transactionId(TRANSACTION_ID)
+                                .resolutions(List.of(
+                                        new Resolution()
+                                                .category(CategoryEnum.RESOLUTION)
+                                                .description("resolution description")
+                                                .date(DATE)
+                                ))
+                                .links(new Links().self(VALID_SELF_LINK)))
+                        .internalData(new InternalData()
+                                .entityId(ENTITY_ID)
+                                .companyNumber(COMPANY_NUMBER)
+                                .deltaAt(VALID_DELTA_AT));
+            }
+
+            public RequestBodyTestArgumentBuilder modifyExternalData(final ExternalData value) {
```

Comment:
Is there any reason that all the setters for this builder have the `modify `prefix`? Throughout the rest of the Thunderbirds code base we have settle on the standard format of using just the property name. The extra 6 characters in the name is just noise and really doesn't add any extra values when using a builder class.

It would be better to change all these setters to remove the `modify `prefix`.

Reply:
Its because they are the opposite of a builder, the constructor builds the default state and the modify methods set the value to null